packages/msw-addon/package.json

```diff
@@ -2,12 +2,27 @@
   "name": "msw-storybook-addon",
   "description": "Mock API requests in Storybook with Mock Service Worker.",
   "version": "1.6.0",
-  "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "main": "./dist/index.browser.js",
+  "types": "./dist/index.browser.d.ts",
+  "exports": {
+    "browser": {
+      "types": "./dist/index.browser.d.ts",
```

Comment:
Also providing the right types since depending on the environment, you will be importing either `msw/browser` or `msw/node` when using `initialize`.